chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -252,24 +251,27 @@
     mode "0640"
   end
 else
-  web_port = 3000
   realm = nil
 end
 
-# Remove rainbows configuration, dating from before the switch to puma
-file "/opt/dell/crowbar_framework/rainbows.cfg" do
-  action :delete
+# Remove rainbows & puma config files, dating from before the switch to passenger
+[
+  "/opt/dell/crowbar_framework/rainbows.cfg",
+  "/etc/sysconfig/crowbar",
+  "/etc/systemd/system/crowbar.service",
+  "/usr/sbin/rccrowbar",
+  "/etc/bluepill/crowbar-webserver.pill",
+  "/etc/init.d/crowbar"
+].each do |old_config|
+  file old_config do
+    action :delete
+  end
 end
 
-template "/etc/sysconfig/crowbar" do
-  source "sysconfig.crowbar.erb"
-  owner "root"
-  group "root"
-  mode "0644"
-  variables(
-    web_host: "127.0.0.1",
-    web_port: web_port,
-  )
+bash "stop old puma process" do
+  code "/usr/bin/pumactl -S /opt/dell/crowbar_framework/tmp/pids/puma.state stop"
+  ignore_failure true
+  only_if { File.exists?("/opt/dell/crowbar_framework/tmp/pids/puma.state") }
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.